smaug/python/ops/nn_ops.py

```diff
@@ -56,11 +56,7 @@ def compute_output_dim(input_dim, weight_dim, stride, padding):
   params = node_pb2.Params()
   params.conv_params.padding = to_padding_type(padding)
   params.conv_params.stride.extend(stride)
-  if activation != None:
-    params.act_params.activation = activation
-    activation_ops.set_activation_params(
-        activation, params.act_params, activation_params)
-
+  activation_ops.to_proto(activation, activation_params, params.act_params)
```

Comment:
It's not clear to me that `params.act_params` is actually a proto.

Reply:
Now it returns a proto.